tools/fast/fastesting/print_writer.go

```diff
@@ -0,0 +1,48 @@
+package fastesting
+
+import (
+	"bufio"
+	"io"
+	"sync"
+	"testing"
+)
+
+type printWriter struct {
+	pw io.WriteCloser
+	wg sync.WaitGroup
+}
+
+func newPrintWriter(t *testing.T) io.WriteCloser {
```

Comment:
Please add some comments on what this method is for, it looks like it pipes the daemons last output to calls to testing.Log